### PR TITLE
Help2guides

### PR DESCRIPTION
**Pull request by jburel**
use omero-guides url instead of help.o.org

This PR replaces https://github.com/ome/omero-documentation/pull/2366

### CODE REVIEW

File: omero/developers/Web/WebGateway.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ rendering images and accessing data on the OMERO server via URLs.
 .. note::
 
     The OMERO.web client also supports URLs linking to specified data in
-    OMERO. See the :help:`OMERO.web user guides <>` for more details.
+    OMERO. See the :help:`OMERO.web User guides <>` for more details.
```
Comment (pwalczysko) [inline omero/developers/Web/WebGateway.rst]:
```suggestion
    OMERO. See the :help:`OMERO.figure User guide <figure/docs/omero_figure.html>` for more details.
```

File: omero/sysadmins/index.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ System Administrator Documentation
 This documentation begins with information aimed at OS-level administrators
 and moves on to day-to-day management of OMERO for facility managers (who may
 find it useful to read the
-:help:`Facility Managers help guide <facility-manager.html>` for an overview
+:help:`Facility Managers section <example_facility_manager.html>` for an overview
```
Comment (pwalczysko) [inline omero/sysadmins/index.rst]:
```suggestion
:help:`Facility Managers walkthrough <example_facility_manager.html>` for an overview
```

The word ``section`` mistakenly suggests that it is about a section within the docs.

File: omero/sysadmins/mapping-restricted-admins.rst [hunk 3]
```diff
@@ -10,7 +10,7 @@ administrator privileges,
 see :doc:`/sysadmins/restricted-admins`.
 The OMERO.web user interface form for creation and editing of
 restricted administrators
-(see the :help:`creating Administrators with restricted privileges <facility-manager.html#lightadmin>` section)
+(see the :help:`creating Administrators with restricted privileges <introduction/docs/group-user-management.html>` section)
```
Comment (pwalczysko) [inline omero/sysadmins/mapping-restricted-admins.rst]:
```suggestion
(see the :help:`Web Interface: Managing Users <introduction/docs/group-user-management.html>` section)
```

File: omero/sysadmins/public.rst [hunk 4]
```diff
@@ -183,10 +183,10 @@ considered "published".
    Project/Dataset/Image hierarchy prepared for the move by the author.
 #. If you have used OMERO.figure to create your figures for publication, you
    can always find the original data by using the 'info' tab, as shown in the
```
Comment (pwalczysko) [inline omero/sysadmins/public.rst]:
```suggestion
   can always find the original data by using the 'info' tab inside the OMERO.figure app, see the
```

File: omero/sysadmins/restricted-admins.rst [hunk 5]
```diff
@@ -28,8 +28,8 @@ still a powerful user so each must be a highly trusted individual.
 
 Full administrators in OMERO can create new administrators with
 restricted privileges using the OMERO.web interface, see the
-:help:`facility managers guide <facility-manager.html#lightadmin>` in our Help
-documentation. OMERO.cli does not yet support easy management of
+:help:`Facility managers section <example_facility_manager.html>` in our User guides.
```
Comment (pwalczysko) [inline omero/sysadmins/restricted-admins.rst]:
```suggestion
:help:`Web Interface: Managing Users <introduction/docs/group-user-management.html>` in our User guides.
```

This leads more directly to the target info.

File: omero/sysadmins/restricted-admins.rst [hunk 6]
```diff
@@ -136,7 +136,7 @@ Client Details:
   executed using OMERO.web or CLI.
 
 - OMERO.web: allows viewing and downloading the data,
-  see :help:`Viewing Data <viewing-data>`.
+  see :help:`Viewing Data <iviewer/docs/index.html>`.
```
Comment (pwalczysko) [inline omero/sysadmins/restricted-admins.rst]:
```suggestion
  see :help:`Viewing Data <iviewer/docs/index.html>` and :help:`Downloading Data <download/docs/index.html>`
```

File: omero/sysadmins/restricted-admins.rst [hunk 7]
```diff
@@ -323,7 +320,7 @@ data and to link the data to those containers or to already
 existing containers owned by the new owner. Since OMERO 5.4.0,
 OMERO.web enables Organizers with :term:`Write Data` privilege
 to create new containers belonging to other users,
-see the :help:`OMERO.web in Data structure <facility-manager#data>`
+see the :help:`OMERO.web in Data structure <introduction/docs/data-management.html>`
```
Comment (pwalczysko) [inline omero/sysadmins/restricted-admins.rst]:
```suggestion
see the :help:`Manage Images in Datasets/Projects <introduction/docs/data-management.html#manage-images-in-datasets-projects>`
```

File: omero/sysadmins/restricted-admins.rst [hunk 8]
```diff
@@ -323,7 +320,7 @@ data and to link the data to those containers or to already
...
 section of our Help documentation.
```
Comment (pwalczysko) [inline omero/sysadmins/restricted-admins.rst]:
```suggestion
section of our User guides.
```

File: omero/sysadmins/restricted-admins.rst [hunk 9]
```diff
@@ -362,7 +359,7 @@ Client Details:
   of ownership (possible only in CLI, will be addressed in the 5.4.x
   series). Creation of Projects, Datasets
   or Screens for other users in OMERO.web is possible since OMERO 5.4.0,
-  see :help:`Data structure (OMERO.web) <facility-manager#data>`.
+  see :help:`Data structure (OMERO.web) <introduction/docs/data-management.html>`.
```
Comment (pwalczysko) [inline omero/sysadmins/restricted-admins.rst]:
```suggestion
  see the :help:`Manage Images in Datasets/Projects <introduction/docs/data-management.html#manage-images-in-datasets-projects>`
```

File: omero/sysadmins/server-permissions.rst [hunk 10]
```diff
@@ -141,7 +141,7 @@ The various permission levels are:
 
 .. seealso::
 
-    :help:`Help guide for sharing data <sharing-data.html>`
+    :help:`Users guide for sharing data <introduction/docs/group-user-management.html>`
```
Comment (pwalczysko) [inline omero/sysadmins/server-permissions.rst]:
```suggestion
    :help:`Users guide for data management and sharing <data-management.html>`
```

File: omero/sysadmins/server-permissions.rst [hunk 11]
```diff
@@ -141,7 +141,7 @@ The various permission levels are:
...
      Workflow guide covering the groups and permissions system
```
Comment (pwalczysko) [inline omero/sysadmins/server-permissions.rst]:
```suggestion
     Workflow guide describing the basics of data management and sharing with links to more detailed permissions documentation
```

File: omero/users/cli/chgrp.rst [hunk 12]
```diff
@@ -24,7 +24,7 @@ How to move data
 ^^^^^^^^^^^^^^^^
 
 * CLI: See below
```
Comment (pwalczysko) [inline omero/users/cli/chgrp.rst]:
```suggestion
* Using CLI: See below
```

File: omero/users/cli/chgrp.rst [hunk 13]
```diff
@@ -24,7 +24,7 @@ How to move data
...
-* :help:`OMERO.web and OMERO.insight<sharing-data#moving>`
+* :help:`OMERO.web <introduction/docs/data-management.html>`
```
Comment (pwalczysko) [inline omero/users/cli/chgrp.rst]:
```suggestion
* Using OMERO.web: See :help:`User guides <introduction/docs/data-management.html>`
```

File: omero/users/clients-overview.rst [hunk 14]
```diff
@@ -20,7 +20,7 @@ and require Java |javaversion_min| (or higher) to be installed on the user's
 computer (this can easily be installed from `<https://java.com/>`_ if it is not
 already included in your OS).
 
-Our user assistance :help:`help website<>` provides a series of
+Our :help:`User guides<>` provide a series of
 workflow-based guides to performing common actions in the client applications,
 such as importing and viewing data, exporting images and using the measuring
```
Comment (pwalczysko) [inline omero/users/clients-overview.rst]:
```suggestion
such as importing and viewing data, exporting images, drawing and analyzing of ROIs.
```

File: omero/users/clients-overview.rst [hunk 15]
```diff
@@ -20,7 +20,7 @@ and require Java |javaversion_min| (or higher) to be installed on the user's
...
 tool.
```
Comment (pwalczysko) [inline omero/users/clients-overview.rst]:
```suggestion
 
```

**Comment by pwalczysko** [comment]
I would replace the text in https://github.com/ome/omero-documentation/pull/2373/files#diff-3e0453ba847735ba6d820656c9c45c921b4beee7a7c8f55f6dbd0ff22e33efa0 line 127:

``Our user assistance :help:`help website <>` features a number of
workflow-based guides to importing, viewing, managing and exporting your data
using OMERO.insight.``

with

``We recommend to use OMERO.insight mainly for imports and concentrate on OMERO.web for all other workflows. See our :help:`User guides <>` for a number of
workflows which show importing data using OMERO.insight and viewing, managing and exporting data
using OMERO.web.``

This has not been changed in this PR, but possibly it is an omission.

**Review by pwalczysko — CHANGES_REQUESTED** [review]
- Couple of commit suggestions 
- Please address https://github.com/ome/omero-documentation/pull/2373#issuecomment-1823095449

**Comment by pwalczysko** [comment]
Also, the build is failing with

14:55:33 (sysadmins/public: line  221) broken    https://omero-guides.readthedocs.io/en/latest/managing-data.html#keyvalue - 404 Client Error: Not Found for url: https://omero-guides.readthedocs.io/en/latest/managing-data.html


I would suggest to replace

:help:`Managing data guide <managing-data.html#keyvalue>`

with

:help:`Annotate User guide <introduction/docs/annotate.html>`

which should fix the error.

**Comment by jburel** [comment]
@pwalczysko it is now green

**Comment by pwalczysko** [comment]
> @pwalczysko it is now green

Not yet, this is the interesting thing, I am watching https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/280/ - rebuilding just now.

File: omero/sysadmins/server-permissions.rst [hunk 16]
```diff
@@ -141,8 +141,8 @@ The various permission levels are:
 
 .. seealso::
 
-    :help:`Help guide for sharing data <sharing-data.html>`
-     Workflow guide covering the groups and permissions system
+    :help:`Users guide for data management and sharing <data-management.html>`
```
Comment (pwalczysko) [inline omero/sysadmins/server-permissions.rst]:
```suggestion
    :help:`Users guide for data management and sharing <introduction/docs/data-management.html>`
```

**Comment by pwalczysko** [comment]
Indeed, one more is broken as reported by Jenkins
17:07:51 (sysadmins/server-permissions: line  144) broken    https://omero-guides.readthedocs.io/en/latest/data-management.html - 404 Client Error: Not Found for url: https://omero-guides.readthedocs.io/en/latest/data-management.html

Suggesting to replace with https://github.com/ome/omero-documentation/pull/2373/commits/fd6ac80257430c28a6e88c35c699f7d7166f592b

**Comment by pwalczysko** [comment]
Now the https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/ is green.

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**